doc/source/visualizing/plots.rst

```diff
@@ -1074,6 +1074,31 @@ negative, we set the scaling to be linear for this field.
    plot.set_log("x-velocity", False)
    plot.save()
 
+Setting axis labels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The axis labels can be manipulated via the
+:meth:`~yt.visualization.profile_plotter.ProfilePlot.set_ylabel` and
+:meth:`~yt.visualization.profile_plotter.ProfilePlot.set_xlabel` functions.  The
+:meth:`~yt.visualization.profile_plotter.ProfilePlot.set_ylabel` function accepts a field name 
+and a string with the desired label. The :meth:`~yt.visualization.profile_plotter.ProfilePlot.set_xlabel`
+function just accepts the desired label and applies this to all of the plots. 
+
+In the following example we create a plot of the average x-velocity and density as a
+function of radius. The xlabel is set to "Radius", for all plots, and the ylabel is set to
+"velocity in x direction" for the x-velocity plot.
+
+.. python-script::
+
+  import yt
+  ds = yt.load("enzo_tiny_cosmology/DD0046/DD0046")
+  ad = ds.all_data()
+  plot = yt.ProfilePlot(ad, "density", ["temperature", "velocity_x"],
+                   weight_field=None)
+   plot.set_xlabel("Radius")
+   plot.set_ylabel("x-velocity", "velocity in x direction")
+   plot.save()
```

Comment:
It looks like these three lines are one space off from the of the lines above. That's causing an error during the doc build:

```
    plot.set_xlabel("Radius")
    ^
IndentationError: unexpected indent
```

Reply:
Ah, nightmare. Apologies!

Reply:
I think I've fixed these now and made the changes from @brittonsmith too

Reply:
What's your e-mail?

Reply:
a.j.kelly@durham.ac.uk